pygmt/datasets/tile_map.py

```diff
@@ -109,10 +109,10 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
     """
     # pylint: disable=too-many-locals
     if contextily is None:
-        raise ModuleNotFoundError(
+        raise ImportError(
             "Package `contextily` is required to be installed to use this function. "
             "Please use `pip install contextily` or "
-            "`conda install -c conda-forge contextily` "
+            "`mamba install -c conda-forge contextily` "
```

Comment:
Will be changed in https://github.com/GenericMappingTools/pygmt/pull/2394, so revert the change here:
```suggestion
            "`conda install -c conda-forge contextily` "
```